frontend/src/app/globals.css

```
@@ -792,6 +792,120 @@ input:focus-visible {
+  height: 100vh;
+  padding: 4rem 1.5rem;
+  overflow: hidden;
+  text-align: center;
+}
+
+html[data-theme="dark"] .notfound-shell {
+  --accent: #4B8BF4;
+  --accent-hover: #6BA2F7;
+  --accent-soft: rgba(75, 139, 244, 0.14);
+  --accent-glow: rgba(75, 139, 244, 0.18);
+  background-color: #0A0D10;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)